docs/user_guide/resource_quota_mgmt.md

```diff
@@ -22,57 +22,81 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-YuniKorn can offer more fine-grained resource quota management comparing to simply
-using namespace resource quota. Here are some how-to documents about setting up
-resource quota management with YuniKorn queues.
+## Quota configuration and rules
+YuniKorn can offer a finer grained resource quota management setup compared to the simple
+namespace resource quota provided by Kubernetes.
 
-## Option 1) Static queues
+Contrary to quotas in Kubernetes YuniKorn does not enforce quotas on submission but on actively consumed resources.
 
-### Goal
+Resource quotas in YuniKorn are linked to the queue and its place in the queue hierarchy.
```

Comment:
*its place -> it's placed

Reply:
it's correct, `its place` is the place of the queue in the hierarchy (possessive form) 
`it's placed`  is the short for of `it is placed` which does not make sense.

https://www.grammarly.com/blog/its-vs-its/